documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -5,11 +5,14 @@
 
 An Aspect Model consists of different BAMM elements: one Aspect and several Properties, Operations, Characteristics, Constraints, Entities, or Events.
 
-An Aspect is created by default when starting an Aspect Model. All other elements can then be added. Every element created in the editor can be altered. 
+An Aspect is created by default when starting an Aspect Model.
+All other elements can then be added.
+Every element created in the editor can be altered.
 
 NOTE: Elements that belong to other namespaces/files cannot be edited.
 
-The naming conventions for Aspect Model elements are defined in the https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/v1.0.0/modeling-guidelines.html#naming-rules[Naming Rules (BAMM 1.0.0)^, opts=nofollow]. The Aspect Model Editor guides you to follow all those BAMM naming conventions for your Aspect Models.
+The naming conventions for Aspect Model elements are defined in the https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/2.0.0/modeling-guidelines.html#naming-rules[Naming Rules (BAMM 2.0.0)^,opts=nofollow].
+The Aspect Model Editor guides you to follow all those BAMM naming conventions for your Aspect Models.
```

Comment:
The Aspect Model Editor guides you to follow all these BAMM naming conventions for your Aspect Models.